phoenix-core/src/main/java/org/apache/phoenix/iterate/OffsetResultIterator.java

```diff
@@ -49,7 +49,7 @@ public Tuple next() throws SQLException {
         while (rowCount < offset) {
             Tuple tuple = super.next();
             if (tuple == null) { return null; }
-            if (isDummy(tuple)) {
```

Comment:
very minor nit:
.size() is probably cheaper than isDummy(), so switching the two predicates may be advised.